src/bind.js

```diff
@@ -18,8 +18,10 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
+function bind(callback, ...elements) {
+  return (...arg) => {
```

Comment:
good, but you can write simply
```suggestion
  return (... arg) => callback(... elements, ... arg);
```